pytorch_lightning/metrics/classification/accuracy.py

```diff
@@ -43,7 +43,8 @@ class Accuracy(Metric):
     Args:
         threshold:
             Threshold probability value for transforming probability predictions to binary
-            `(0,1)` predictions, in the case of binary or multi-label inputs.
+            (0,1) predictions, in the case of binary or multi-label inputs. If not set it
+            defaults to 0.5.
```

Comment:
```suggestion
            `(0,1)` predictions, in the case of binary or multi-label inputs.
```
no need to repeat defaults

Reply:
So the new default for threshold is `None`, but that acts like 0.5 if not set and has to be used (like the `top_k` parameter)

Reply:
I see, but some other functions like `_input_format_classification` have set value and also write defaults...
what is the reason/advantage if having default None and not use value 0.5 as it is used anyway?